### DISCUSSION TITLE
我不理解为什么屏蔽了一些网站

### DISCUSSION THREAD

**Discussion by SharpIceX** [eallion/uBlacklist-subscription-compilation#47] [General]
我目前发现屏蔽了 掘金、cnblog、NGA玩家社区、恩山无线论坛、baiyun-hotel.com 等等，我不太理解为什么屏蔽，以及最后一个能被屏蔽我更加不理解了

**Comment by eallion** [comment]
这些屏蔽列表全部来自上游，这个列表只作了合并处理。另：这些网站的目标用户可以通过输入 URL 直接进入网站，还需要搜索进入的就不算目标用户。（就我个人来说，有 2 个网站被屏蔽一点都不无辜。）

我稍后还是检查一下具体是哪一个列表屏蔽了这些网站。

**Reply by SharpIceX** [reply]
我在规则列表里找不到baiyun-hotel.com的痕迹但是确实把baiyun-hotel.com屏蔽了太奇怪了。。。

**Reply by eallion** [reply]
![PixPin_2024-07-24_13-27-02](https://github.com/user-attachments/assets/48affb14-f646-4172-adfb-ed4240749c5a)

不是域名屏蔽，应该是触发了其他关键词。

**Reply by eallion** [reply]
https://github.com/eallion/uBlacklist-subscription-compilation/commit/5ca7e4558cf25ea77f5f72e470743712d91dd327
